pyproject.toml

```diff
@@ -0,0 +1,32 @@
+[tool.poetry]
+name = "tiff-2-png"
+version = "0.1.0"
+description = "Tool to convert stack of tiff images to a png texture map"
+authors = ["Camilo Diaz <camilo_diaz@brown.edu>"]
+license = "MIT"
+
+packages = [
+    { include = "tiff_2_png" },
+]
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
```

Comment:
Have you tested this? In my experience, a lot changes from minor version to minor version of Python

Reply:
So you may want to stick to something closer to what you are using

Reply:
The whole convert to png pipeline works (at least on my machine).
That line was auto added after executing the command: `poetry add opencv-contrib-python`